.env.development

```
@@ -6,6 +6,7 @@
 
 # feature flags
 VITE_FEATURE_THORCHAIN_TCY_ACTIVITY=true
+VITE_FEATURE_QUICK_BUY=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_FEATURE_QUICK_BUY key should go before the VITE_FEATURE_THORCHAIN_TCY_ACTIVITY key

(UnorderedKey)